cobigen-cli/src/main/java/com/devonfw/cobigen/cli/constants/MessagesConstants.java

```diff
@@ -58,4 +58,10 @@
     public static final String TEMPLATES_OPTION_DESCRIPTION =
         "List of templates that will be generated. They need to be specified with numbers separated by comma.";
 
+    /**
+     * Message constant: description of the plug-in update
+     */
+    public static final String UPDATE_OPTION_DESCRIPTION =
+        "Using update command compare maven central plugin and update.";
```

Comment:
`Use update command to find out which plug-ins are outdated and choose which ones do you want to update.`